src/components/__tests__/CustomDisciplineMenuItem.spec.tsx

```diff
@@ -21,6 +21,7 @@ describe('Components', () => {
       icon: 'none',
       numberOfChildren: 1,
       isLeaf: false,
+      isFirstLevel: true,
```

Comment:
I'm not happy with this naming. As we have `isLeaf` i would suggest to call it `isRoot`. Otherwise we have to name it `isFirstLevel` and `isLastLevel`, but as we have a classic tree i would strongly prefer the first.

Reply:
I am fine with it. The reason I decided against it was that for me the root was the home screen which has multiple `Bereiche`. But I guess `isRoot` is way more intuitive :+1: